doc/release/0.18.0-notes.rst

```diff
@@ -65,18 +70,124 @@ zero-phase low-pass FIR filter, and downsamples using `scipy.signal.upfirdn`.
+
+
+Discrete-time linear systems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`scipy.signal.dlti` provides an implementation of discrete-time linear systems.
+Accordingly, the `StateSpace`, `TransferFunction` and `ZerosPolesGain` classes
+have learned a the new keyword, `dt`, which can be used to create discrete-time
+instances of the corresponding system representation.
+
+
+`scipy.sparse` improvements
 -----------------------------------------
 
 The functions `sum`, `max`, `mean`, `min`, `transpose`, and `reshape` in
 `scipy.sparse` have had their signatures augmented with additional arguments
 and functionality so as to improve compatibility with analogously defined
 functions in `numpy`.
 
+Sparse matrices now have a `count_nonzero` method, which counts the number of
+nonzero elements in the matrix. Unlike `getnnz()` and ``nnz`` propety,
+which return the number of stored entries (the length of the data attribute),
+this method counts the actual number of non-zero entries in data.
+
+
+`scipy.optimize` improvements
+-----------------------------
+
+The implementation of Nelder-Mead minimization,
+`scipy.minimize(..., method="Nelder-Mead")`, obtained a new keyword,
+`initial_simplex`, which can be used to specify the initial simplex for the
+optimization process.
+
+Initial step size selection in CG and BFGS minimizers has been improved. We
+expect that this change will improve numeric stability of optimization in the
+vast majortiy of use cases. See pull request #5536 for details.
+
+Handling of infinite bounds in SLSQP optimization has been improved. We expect
+that this change will improve numeric stability of optimization in the vast
+majortiy of use cases. See pull request #6024 for details.
+
+A large suite of global optimization benchmarks has been added to 
+``scipy/benchmarks/go_benchmark_functions``. See pull request #4191 for details.
+
+
+`scipy.stats` improvements
+--------------------------
+
+Trapezoidal distribution has been implemented as `scipy.stats.trapz`.
+Skew normal distribution has been implemented as `scipy.stats.skewnorm`.
+Burr type XII distribution has been implemented as `scipy.stats.burr12`. 
+
+Random matrices
+~~~~~~~~~~~~~~~
+
+`scipy.stats.special_ortho_group` and `scipy.stats.ortho_group` provide
+generators of random matrices in the SO(N) and O(N) groups, respectively. They
+generate matrices in the Haar distribution, the only uniform distribution on
+these group manifolds.
+
+`scipy.stats.random_correlation` provides a generator for random
+correlation matrices, given specified eigenvalues.
+
+
+`scipy.linalg` improvements
+---------------------------
+
+`scipy.linalg.svd` gained a new keyword argument, ``lapack_driver``. Available
+drivers are ``gesdd`` (default) and ``gesvd``.
+
+`scipy.linalg.lapack.ilaver` returns the version of the LAPACK library SciPy
+links to.
+
+
+`scipy.spatial` improvements
+----------------------------
+
+Boolean distances, `pdist`, has been sped up. Improvements vary by the function
```

Comment:
double backticks, this won't resolve as a link. (or `` `scipy.spatial.pdist` `` for a link)